bin/README.md

```diff
@@ -0,0 +1,8 @@
+#### Local Tinlake deployment default keys
+```
```

Comment:
The idea is to have default keys which you only need to import once into Metamask for local UI testing.